build/Signing.proj

```diff
@@ -106,6 +106,10 @@
       <FilesToSign Include="$(PackagesDirectory)/**/*.cab">
         <Authenticode>$(InternalCertificateId)</Authenticode>
       </FilesToSign>
+      <FilesToSign Include="$(PackagesDirectory)/**/*.nupkg"
+                   Exclude="$(PackagesDirectory)/**/*symbols.nupkg">
+        <Authenticode>$(InternalCertificateId)</Authenticode>
```

Comment:
This should be 'NuGet'

Reply:
@mmitche 
"This should be 'NuGet'" - what should be 'NuGet'? not the file names; they are : 'NuPkg'.

Reply:
The cert name